telescope/utils.py

```diff
@@ -299,6 +299,19 @@ class BugTracker:
     def __init__(self, cache=None):
         self.cache = cache
 
+    async def ping(self) -> bool:
+        """
+        Returns True if we can succesfully hit and parse the /rest/version endpoint.
+        """
+        url = f"{config.BUGTRACKER_URL}/rest/version"
```

Comment:
I would suggest to use a endpoint that verifies the provided API key `config.BUGTRACKER_API_KEY`. Like this one? https://github.com/mozilla/jira-bugzilla-integration/blob/3cd50288b468cc76e0a1c42884368d6ef4e6d577/jbi/bugzilla/client.py#L65-L72

Reply:
Ah that's a good call. Adjusting